### ISSUE TITLE
Codelense on light theme is less than 4.5:1.

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#126899]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_VSCode;#A11ySev3;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#ColorContrast;#WCAG1.4.3;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;

![2717_A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectSubscription_ColorContrast](https://user-images.githubusercontent.com/35271042/122933740-c22d5900-d323-11eb-812b-060182383f3e.png)

### Environment details:
Application Name: Azure Stream Analytics Tools for Visual Studio Code
Windows Version: Win10 
Visual studio code  version : -1.55.1(User Setup)

### Repro Steps:

1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension . 
2. Tab to "Explorer" section in activity bar. 
3. Press control +shift +P and edit bar will appear. 
4. Type "ASA:Create new project" and press enter to create project. 
5. New project will get created. 
6. Tab to Input item in Project Tree Section 
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter. 
8. Select IoT hub from available option using enter key. 
9. Select "select from azure subscriptions" from available option using enter key. 
10. Type any name for input file and press enter to create Input json file. 
11. Tab to "Select Subscription" button.
12. Start AI4D tool and check the color contrast ratio of select subscription text.

### Actual Result:
The color contrast ratio of the text "Select Subscription" button available in input json file is less than 4.5:1.

Note: - This issue is also observed with "Select IoT Hub", "Select Shared Access Policy name" text

### Expected Result:
The color contrast ratio of the text "Select Subscription" button available in input json file should be greater than or equal to 4.5:1.

### User Impact:
The low vision users may not able to see the "Select Subscription" text available in the input json file.

### Attachment:
[A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectSubscription_ColorContrast.zip](https://github.com/microsoft/vscode/files/6693769/A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectSubscription_ColorContrast.zip)

**Comment by misolori** [comment]
We'd have to use something that's at least as strong as `#767676`:

<img width="406" alt="CleanShot 2021-06-22 at 06 36 38@2x" src="https://user-images.githubusercontent.com/35271042/122934292-41229180-d324-11eb-82aa-cf87e2e09406.png">

<img width="480" alt="CleanShot 2021-06-22 at 06 37 28@2x" src="https://user-images.githubusercontent.com/35271042/122934378-539ccb00-d324-11eb-9e8e-65731fe453e4.png">

**Comment by isidorn** [comment]
@prasadaps thanks for creating this issue.

Since the code lens is designed to be not annoying, I suggest that we go with the minimum required contrast of 3:1.
Not the recommend contrast of 4.5:1
More about required contrast ratios can be found [here](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=459a1bff-f94b-4be7-a66e-6eeb0d846c29)

@misolori can you maybe find a less strong color that passes 3:1 contrast ratio and we see how that looks?

**Comment by misolori** [comment]
I've tweaked the light theme colors to be at least 3:1, so going from `#999999` => `#919191`:

<img width="324" alt="CleanShot 2021-06-28 at 11 28 18@2x" src="https://user-images.githubusercontent.com/35271042/123685869-fe6d2780-d803-11eb-8ec1-00ac4cbf0f32.png">
